Something still seems to be failing
```
./scripts/run-compat.sh "sepolia-ecotone"
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404 
make: *** [Makefile:68: verify-sepolia-ecotone] Error 22
```

--- a/op-program/Makefile
+++ b/op-program/Makefile
@@ -67,7 +67,7 @@ verify-mainnet-genesis: op-program-host op-program-client
 verify-sepolia-ecotone: op-program-host op-program-client
 	./scripts/run-compat.sh "sepolia-ecotone"
 
-verify-compat: verify-sepolia-delta verify-mainnet-genesis
+verify-compat: verify-sepolia-delta verify-sepolia-ecotone verify-mainnet-genesis
 
 .PHONY: \
 	op-program \

--- a/op-program/scripts/run-compat.sh
+++ b/op-program/scripts/run-compat.sh
@@ -5,7 +5,7 @@ SCRIPTS_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 COMPAT_DIR="${SCRIPTS_DIR}/../temp/compat"
 
 TESTNAME="${1?Must specify compat file to run}"
-BASEURL="${2:-https://github.com/ethereum-optimism/chain-test-data/releases/download/2024-03-14}"
+BASEURL="${2:-https://github.com/ethereum-optimism/chain-test-data/releases/download/2024-03-14.3}"
 
 URL="${BASEURL}/${TESTNAME}.tar.bz"
 

